pom.xml

```diff
@@ -990,8 +990,9 @@ under the License.
 							<groupId>org.apache.maven.plugins</groupId>
 							<artifactId>maven-javadoc-plugin</artifactId>
 							<configuration>
-								<additionalJOptions>
+								<additionalJOptions combine.children="append">
```

Comment:
this isn't necessary; it's already set up in the main javadoc-plugin config.

Reply:
Alright - saw it bus wasn't sure that it's enough to have it only once...I'll adapt the PR